example/src/App.js

```diff
@@ -65,6 +66,19 @@ function App() {
     setAnchorEl(null);
   };
 
+  if (process.env.REACT_APP_GOOGLE_ANALYTICS) {
+    ReactGA.initialize(process.env.REACT_APP_GOOGLE_ANALYTICS);
+    // We disable this check since, while the hooks are called conditionally,
+    // it is safe to do so because this is not a condition that can change
+    // between renders. `process.env` becomes a static object.
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const location = useLocation();
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    useEffect(() => {
+      ReactGA.pageview(location.pathname + location.search);
+    }, [location]);
+  }
+
```

Comment:
Would this work?

```suggestion
  if (process.env.REACT_APP_GOOGLE_ANALYTICS) {
    ReactGA.initialize(process.env.REACT_APP_GOOGLE_ANALYTICS);
  }
  const location = useLocation();
  useEffect(() => {
    if (process.env.REACT_APP_GOOGLE_ANALYTICS) {
      ReactGA.pageview(location.pathname + location.search);
    }
  }, [location]);
```
We'd be unnecessarily calling `useLocation` and `useEffect` when not enabled, but that maybe that's acceptable here?